src/frontend/apps/standalone_site/src/features/Contents/features/ClassRoom/components/Read/ClassRoomItem.tsx

```diff
@@ -1,34 +1,83 @@
-import { Text, Box } from 'grommet';
+import { Text, Box, CheckBox } from 'grommet';
 import { FormSchedule, InProgress } from 'grommet-icons';
 import {
   ClassroomLite,
   StyledLink,
   ContentCard,
   TextTruncated,
 } from 'lib-components';
-import { Fragment } from 'react';
+import { Fragment, useEffect, useState } from 'react';
 import { useIntl } from 'react-intl';
 
 import { ReactComponent as VueListIcon } from 'assets/svg/iko_vuelistesvg.svg';
 import { ReactComponent as ClassroomsIcon } from 'assets/svg/iko_webinairesvg.svg';
+import { useSelectFeatures } from 'features/Contents/store/selectionStore';
 import { routes } from 'routes';
 import { localDate } from 'utils/date';
 
 const ClassRoom = ({ classroom }: { classroom: ClassroomLite }) => {
   const intl = useIntl();
   const classroomPath = routes.CONTENTS.subRoutes.CLASSROOM.path;
 
+  const { isSelectionEnabled, selectedItems, setSelectedItems } =
+    useSelectFeatures();
+  const [isClassroomSelected, setIsClassroomSelected] = useState<boolean>(
+    () => selectedItems.includes(classroom.id) || false,
+  );
+
+  const onCardClick = () => {
+    if (isSelectionEnabled) {
+      if (isClassroomSelected) {
+        setSelectedItems(
+          selectedItems.filter((videoId) => videoId !== classroom.id),
+        );
+        setIsClassroomSelected(false);
+      } else {
+        setSelectedItems([...selectedItems, classroom.id]);
+        setIsClassroomSelected(true);
+      }
+    }
+  };
```

Comment:
IMO, you can put this function in your selectionStore, you will have it for all your contents.

Reply:
I can't because I use the `isClassroomSelected` property which is a local state

Reply:
I find it quite weird because it uses the local state `isClassroomSelected` and the `setIsClassroomSelected` method which is alreadty in the store, I can't really figure out how to make it work nicely in a better way than this